src/Config.php

```diff
@@ -34,25 +35,26 @@ class Config extends AbstractConfig
      * Static method for loading a Config instance.
      *
      * @param  string|array $path
+     * @param  boolean      $prefix
      *
      * @return Config
      */
-    public static function load($path)
+    public static function load($path, $prefix = false)
     {
-        return new static($path);
+        return new static($path, $prefix);
     }
 
     /**
      * Loads a supported configuration file format.
      *
      * @param  string|array $path
+     * @param  boolean      $prefix
      *
      * @throws EmptyDirectoryException    If `$path` is an empty directory
      */
-    public function __construct($path)
+    public function __construct($path, $prefix = false)
```

Comment:
Can you please use an associative array with the configuration instead of having `$prefix` directly as second parameter?

Reply:
Just to clarify, the constructor method should look like `__construct($path, $options = [])` where `prefix` would be a key under `$options`.